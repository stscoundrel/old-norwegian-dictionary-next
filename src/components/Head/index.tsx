import NextHead from 'next/head'

export default function Head() {
  // Populate per site/page logic.
  const title = ''
  const description = ''
  const canonicalUrl = ''
  const siteName = ''

  return (
    <NextHead>
      <link rel='icon' href='/favicon.ico' />
        <title>{title}</title>
        <link rel='icon' type='image/png' sizes='48x48' href='/favicon-48x48.png' />
        <link rel='icon' type='image/png' sizes='32x32' href='/favicon-32x32.png' />
        <link rel='icon' type='image/png' sizes='16x16' href='/favicon-16x16.png' />
        <link rel="apple-touch-icon" href="/favicon-96x96.png"></link>
        <meta
          name='description'
          content={description}
        />
        <meta name='og:title' content={title} />
        <meta
          name='og:description'
          content={description}
        />
        <meta
          property='og:site_name'
          content={siteName}
        />
        <meta
            property='og:url'
            content={canonicalUrl}
          />
        <meta
          property='og:locale'
          content='en'
        />
        <meta
          property='og:type'
          content='website'
        />
        <meta name='twitter:card' content='summary_large_image' />
        <meta name='twitter:title' content={title} />
        <meta
          name='twitter:description'
          content={description}
        />

        <link rel="canonical" href={canonicalUrl} />

        <meta name="theme-color" content="#3b4f68" />
        <link rel="manifest" href="/manifest.json" />
        <meta name="viewport" content="initial-scale=1.0, width=device-width" />
    </NextHead>
  )
}
