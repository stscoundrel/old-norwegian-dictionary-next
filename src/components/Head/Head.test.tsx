import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import Head from './index'

describe('Head component', () => {
  test('Does not crash', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Head />, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  test('Matches snapshot', () => {
    const tree = renderer.create(<Head />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
