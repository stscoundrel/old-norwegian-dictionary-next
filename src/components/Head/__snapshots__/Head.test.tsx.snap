// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Head component Matches snapshot 1`] = `null`;
